app/templates/frameworks/_framework_actions.html

```diff
@@ -24,128 +24,80 @@
-      <strong>Your company details were saved</strong>
-    </p>
-  </div>
-  {% endif %}
-</li>
-
-<li class="browse-list-item">
-  {% if not application_company_details_confirmed %}
-    <h2>2. Make supplier declaration</h2>
-    <p class="instruction-list-item-box inactive">Can't start yet</p>
-  {% else %}
-    {% if declaration_status == 'unstarted' %}
-    <a class="browse-list-item-link" href="{{ url_for('.framework_start_supplier_declaration', framework_slug=framework.slug) }}">
-      <h2>2. Make supplier declaration</h2>
-    </a>
+ {% if application_made %}
+    {% if counts.draft %}
+      {{ dmAlert({
+        "titleText": "Your application is complete, and will be automatically submitted.",
+        "text": "You still have {count} unsubmitted draft {service_string}. You can edit or remove draft services at any time before the deadline.".format(count=counts.draft, service_string=pluralize(counts.draft, 'service', 'services')),
+        "type" : "success"
+        })
+      }}
     {% else %}
-    <a class="browse-list-item-link" href="{{ url_for('.framework_supplier_declaration_overview', framework_slug=framework.slug) }}">
-      <h2>2. Edit supplier declaration</h2>
-    </a>
-    {% endif %}
-    <p class="browse-list-item-body">
-      Confirm eligibility, agree to the application terms and provide supplier&nbsp;information.
-    </p>
-    {% if declaration_status == 'unstarted' and not counts.complete %}
-      <div class="browse-list-item-status-quiet">
-        <p class="browse-list-item-status-title">You need to make the supplier declaration</p>
-      </div>
-    {% elif declaration_status == 'started' and not counts.complete %}
-      <div class="browse-list-item-status-quiet">
-        <p class="browse-list-item-status-title">You need to finish making the supplier declaration</p>
-      </div>
-    {% elif declaration_status == 'unstarted' and counts.complete %}
-      <div class="browse-list-item-status-angry">
-        <p class="browse-list-item-status-title">
-          <strong>No services will be submitted because you haven’t made the supplier declaration</strong>
-        </p>
-      </div>
-    {% elif declaration_status == 'started' and counts.complete %}
-      <div class="browse-list-item-status-angry">
-        <p class="browse-list-item-status-title">
-          <strong>No services will be submitted because you haven’t finished making the supplier declaration</strong>
-        </p>
-      </div>
-    {% elif declaration_status == 'complete' %}
-      {% if counts.complete and application_company_details_confirmed %}
-        <div class="browse-list-item-status-happy">
-      {% else %}
-        <div class="browse-list-item-status-default">
-      {% endif %}
-        <p class="browse-list-item-status-title">
-          <strong>You’ve made the supplier declaration</strong>
-        </p>
-      </div>
+      {{ dmAlert({
+        "titleText": "Your application is complete, and will be automatically submitted.",
```

Comment:
Suggest amend: Your application is complete and will be submitted automatically.